### DISCUSSION TITLE
[docs example confusion report] Dark vs. light image examples shouldn't look the same

### DISCUSSION THREAD

**Discussion by webknjaz** [pradyunsg/furo#835] [Potential Issue] [answered]
I was just showing https://pradyunsg.me/furo/reference/images/#different-images-for-dark-light-mode to a tech writer and wanted to demonstrate that toggling the mode at the top of that web page changes images.
I was expecting that the difference would be visible, but the images looked the same.
It took a minute to figure out the confusion — the image URLs are the same and the image is not different in any of the variants of this example.

I propose switching the example image to be different for dark vs. light modes. And additionally, it'd be good to use different images for `image` vs. `figure`.

**Comment by pradyunsg** [comment, marked as answer]
https://github.com/pradyunsg/furo/issues/836

Definitely worth fixing.
